schematic/store/synapse.py

```diff
@@ -2025,11 +2027,78 @@ def upsertTable(synStore, tableToLoad: pd.DataFrame = None, tableName: str = Non
 
         synConfig = SynapseConfig(username, authtoken, synStore.getDatasetProject(datasetId))
         synapseDB = SynapseDatabase(synConfig)
-        synapseDB.upsert_table_rows(table_name=tableName, data=tableToLoad)
+
+        try:
+            # Try performing upsert
+            synapseDB.upsert_table_rows(table_name=tableName, data=tableToLoad)
+        except(SynapseHTTPError) as ex:
+            # If error is raised because Table has old `Uuid` column and not new `Id` column, then handle and re-attempt upload
+            if 'Id is not a valid column name or id' in str(ex):
+                TableOperations._update_table_uuid_column(synStore, existingTableId)
+                synapseDB.upsert_table_rows(table_name=tableName, data=tableToLoad)
+            # Raise if other error
+            else:
+                raise ex
 
         return existingTableId
 
-    def updateTable(synStore, tableToLoad: pd.DataFrame = None, existingTableId: str = None,  update_col: str = 'Uuid',  restrict: bool = False):
+    def _update_table_uuid_column(synStore: SynapseStorage, table_id: str) -> None:
```

Comment:
@GiaJordan I know this would be annoying but I think we should check the model to see if the model itself is generating a UUID column before deleting, just as a precaution.

Reply:
Yes I think so. I don't think we want to ban people from having a Uuid column in case they feel like they need one for whatever reason.

Reply:
@mialy-defelice do you know what happens now if the schema is generating a `Uuid` column? From what I see it only creates a new `Uuid` column if one isn't present, and only fills in uuid values if there isnt something already there. Is that correct?

Reply:
@mialy-defelice is there a way to see if a given column is in the schema besides the `is_class_in_schema` method from the schema explorer?